test/mocks/service/fee_service_mock.go

```diff
@@ -26,3 +26,8 @@ func (mfs *MockFeeService) CalculateFee(token string, amount int64) (fee, remain
 	args := mfs.Called(token, amount)
 	return args.Get(0).(int64), args.Get(1).(int64)
 }
+
+func (mfs *MockFeeService) CalculatePercentageFee(amount int64, feePercentage int64) (fee, remainder int64) {
+	// TODO implement me
+	panic("implement me")
```

Comment:
why in todo?

Reply:
The method is generated by GoLand IDE to respect Fee interface. Currently body is not necessary. The TODO comment is already removed